src/factory/Layer.js

```diff
@@ -86,6 +106,61 @@ export const LayerFactory = {
     return layer;
   },
 
+  /**
+   * Returns an OpenLayers WFS layer instance due to given config.
+   *
+   * @param  {Object} lConf  Layer config object
+   * @return {ol.layer.Tile} OL WFS layer instance
+   */
+  createWfsLayer: function (lConf) {
+    // set a default projection if not set in config
+    if (!lConf.projection) {
+      lConf.projection = 'EPSG:3857';
```

Comment:
Shouldn't we assume the Map projection if no projection is specified for the WFS Layer?

Reply:
Yes you are right, using the map's SRS is more reliable.